docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md

```diff
@@ -0,0 +1,125 @@
+# Dasharo Compatibility: Logo customization functionality
+
+## Test cases common documentation
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup/#firmware).
+1. Proceed with the
+    [Generic test setup: OS installer](../../generic-test-setup/#os-installer).
+1. Proceed with the
+    [Generic test setup: OS installation](../../generic-test-setup/#os-installation).
+1. Proceed with the
+    [Generic test setup: OS boot from disk](../../generic-test-setup/#os-boot-from-disk).
```

Comment:
In this section, we should add information, that familiarizing with the logo replacement procedure is necessary.

Reply:
Logo replacement procedure is mentioned later in test steps as it is main part of the test